app/assets/javascripts/modules/remove-filter.js

```diff
@@ -55,38 +54,37 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
          * Just removing " beta " from the haystack would result in
          * "alphagamma", which is wrong.
          */
-        var haystack = ' ' + currentVal.trim() + ' ';
-        var needle = ' ' + decodeEntities(removeFilterValue.toString()) + ' ';
-        var newVal = haystack.replace(needle, ' ').replace(/  /g, ' ').trim();
+        var haystack = ' ' + currentVal.trim() + ' '
+        var needle = ' ' + decodeEntities(removeFilterValue.toString()) + ' '
+        var newVal = haystack.replace(needle, ' ').replace(/ {2}/g, ' ').trim()
 
-        $input.val(newVal).trigger(onChangeSuppressAnalytics);
-      }
-      else if (elementType == 'OPTION') {
-        $('#' + removeFilterFacet).val('').trigger(onChangeSuppressAnalytics);
+        $input.val(newVal).trigger(onChangeSuppressAnalytics)
+      } else if (elementType === 'OPTION') {
+        $('#' + removeFilterFacet).val('').trigger(onChangeSuppressAnalytics)
       }
     }
 
-    function getInput(removeFilterName, removeFilterValue, removeFilterFacet) {
-      var selector = (!!removeFilterName) ? " input[name='" + removeFilterName + "']" : " [value='" + removeFilterValue + "']";
+    function getInput (removeFilterName, removeFilterValue, removeFilterFacet) {
+      var selector = (removeFilterName) ? " input[name='" + removeFilterName + "']" : " [value='" + removeFilterValue + "']"
```

Comment:
Can you explain the removal of the `!!` please?

Reply:
`standard --fix` did this. I guess it knows that `removeFilterName` is truthy or falsy, even if it's not explicitly a `boolean`, and so casting it via `!!` is unnecessary...?

Reply:
Yeah, I don't know. Although I guess if all the tests pass it should be fine, I just wanted to check.